lib/api.js

```diff
@@ -652,14 +652,25 @@ api.declare({
   title: 'See the list of AMIs and their usage',
   stability: API.stability.experimental,
   scopes: [['ec2-manager:internals']],
-  description: 'Lists out all of the AMIs and their usage',
+  description:
+    [
+      'List AMIs and their usage by returning a list of objects in the form:',
+      '{',
+      [
+        'region: string',
+        'volumetype: string',
+        'lastused: timestamp',
+      ].join('\n\t'),
+      '}',
+    ].join('\n'),
```

Comment:
:heart:

Reply:
Would this be neater with the `` query strings?